ontology/yaml/resources/SAFETY/entity_types/LDS.yaml

```diff
@@ -85,3 +85,11 @@ LDS_WLA_BCM:
   opt_uses:
   - water_leak_cable_distance_sensor
 
+SWV_CM:
```

Comment:
If this is not an LDS then it doesnt belong in the LDS file. Please move to SWV.yaml.

Reply:
sir, I found VLV for valves is in HVAC, so I moved this to VLV.yaml and will now define SWCM for water solenoid valves in abstract. ultimately will use VLV_SWCM for sommand and status monitoring. Thanks